libs/testnorge-domain/src/main/java/no/nav/registre/testnorge/domain/dto/arena/testnorge/vedtak/NyttVedtakResponse.java

```diff
@@ -24,7 +24,7 @@
     private List<NyttVedtakAap> nyeRettigheterAap;
 
     @JsonAlias({
-            "nyeTiltaksdeltakelse", "nyeTiltakspenger", "nyeTilleggsytelser"
+            "nyeTiltaksdeltakelse", "nyeTiltakspenger", "nyeTilleggsytelser", "nyeEndreDeltakerstatus"
```

Comment:
Skal endreDeltakerstatus være under NyttVedtakTiltak? I så fall kan du legge de nye feltene som er i NyttVedtakEndreDeltakerstatus inn i NyttVedtakTiltak, og fjerne klassen NyttVedtakEndreDeltakerstatus. 

Et annet alternativ er å ha den egne klassen NyttVedtakEndreDeltakerstatus, men da foreslår jeg at den extender NyttVedtak for å arve noen av fellesfeltene derfra (f.eks finnes avbruddKode og saksbehandler allerede i NyttVedtak). Da må du legge til en ny liste i denne klassen, noe sånt som:
`@JsonAlias({"nyeEndreDeltakerstatus"})
private List<NyttVedtakEndreDeltakerstatus> nyeRettigheterEndreDeltakerstatus;` 

eller liknende.

Reply:
Lurer på om det greieste kanskje er den første løsningen jeg nevnte, å legge de nye feltene inn i NyttVedtakTiltak... Da slipper du litt hazzle med å endre alle klasser som extender RettighetRequest

Reply:
Gikk for første løsningen, koden skal være oppdatert nå :)